cloud/deploy/deploy.go

```diff
@@ -143,15 +143,22 @@ func Deploy(path, deploymentID, wsID, pytest, envFile, imageName string, prompt
 	return nil
 }
 
-func getDeploymentInfo(deploymentID, wsID string, prompt bool, cloudDomain string, client astro.Client) (deploymentInfo, error) {
+func getDeploymentInfo(deploymentID, wsID, deploymentName string, prompt bool, cloudDomain string, client astro.Client) (deploymentInfo, error) {
 	// Use config deployment if provided
 	if deploymentID == "" {
 		deploymentID = config.CFG.ProjectDeployment.GetProjectString()
+		if deploymentID != "" {
+			fmt.Printf("Deployment ID found in the config file. This Deployment ID will be used for the deploy\n")
+		}
+	}
+
+	if deploymentID != "" && deploymentName != "" {
```

Comment:
nitpick: shouldn't we only rely on deploymentID from config only in the case when a user does not provide deploymentID or deploymentName in the command itself? This might ensure priority order remains consistent throughout CLI, as values passed as flags or arguments would be preferred over configured values.

Reply:
I'm not really sure of how this currently works

Reply:
looking at the code deploymentId is only got from the config if the user does not supply a deployment id